### PR TITLE
NEXUS-8900 metrics functional tests

### PR DESCRIPTION
**Pull request by bradbeck**
https://issues.sonatype.org/browse/NEXUS-8900

Replaces: https://github.com/sonatype/nexus-oss/pull/1483

### CODE REVIEW

**Comment by bradbeck** [comment]
http://bamboo.s/browse/NX3-OSSF476-1

**Comment by bradbeck** [comment]
http://bamboo.s/browse/NX3-OSSF476-5

**Comment by mcculls** [comment]
+1 looks ok to me

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Metrics.js [hunk 1]
```diff
@@ -22,11 +22,12 @@ Ext.define('NX.coreui.controller.Metrics', {
   extend: 'Ext.app.Controller',
   requires: [
     'Ext.Ajax',
+    'NX.I18n',
     'NX.Messages',
     'NX.Permissions',
-    'NX.util.Url',
+    'NX.coreui.view.support.Metrics',
```
Comment (jdillon) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Metrics.js]: this should be uneeded, views MVC configuration below implies this requires.

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/support/Metrics.js [hunk 2]
```diff
@@ -203,24 +204,33 @@ Ext.define('NX.coreui.view.support.Metrics', {
     ]
   },
 
+  loadStoreByQuery: function (data, query) {
```
Comment (jdillon) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/support/Metrics.js]: this probably should be marked as @private ?

**Pull request merged**